tests/acceptance/run.sh

```diff
@@ -82,7 +82,16 @@ then
 	done
 elif [[ -n  "${RUN_PART}" ]]
 then
-	ALL_SUITES=$(find "${FEATURES_DIR}"/ -type d | sort | rev | cut -d"/" -f1 | rev)
+  	ALL_SUITES=$(find "${FEATURES_DIR}"/ -type d | sort | rev | cut -d"/" -f1 | rev | grep -v '^[[:space:]]*$')
```

Comment:
@saw-jan please write some explanation here in GitHub about why you needed to add the `grep -v '^[[:space:]]*$'` to the end of this command pipeline.

Also, the indent has changed - but that is not a big problem.

Reply:
`grep -v '^[[:space:]]*$'` removes empty new lines